### PR TITLE
Feat(Query): Healthcheck Not Set for docker compose

### PR DESCRIPTION
**Pull request by cxAndreFelicidade**
**Proposed Changes**
- Query: Healthcheck Not Set for docker compose

I submit this contribution under the Apache-2.0 license.

### CODE REVIEW

File: assets/queries/dockerCompose/healthcheck_not_set/query.rego [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+}
+
+check_h_parameters(hcheck)
+{
+    not common_lib.valid_key(hcheck,"interval")
+}else{
+    not common_lib.valid_key(hcheck,"timeout")
+}else{
+    not common_lib.valid_key(hcheck,"retries")
+}
```
Comment (rafaela-soares) [inline assets/queries/dockerCompose/healthcheck_not_set/query.rego:59]: Not sure if we need this policy

**Review by rafaela-soares — CHANGES_REQUESTED** [review]

**Review by rafaela-soares — APPROVED** [review]
LGTM 🚀

File: assets/queries/dockerCompose/healthcheck_not_set/test/positive_expected_result.json [hunk 2]
```diff
@@ -0,0 +1,20 @@
+[
+    {
+      "queryName": "Healthcheck Not Set",
+      "severity": "MEDIUM",
+      "line": 4,
+      "filename": "positive1.yaml"
```
Comment (joaorufi) [inline assets/queries/dockerCompose/healthcheck_not_set/test/positive_expected_result.json:6]:
```suggestion
      "fileName": "positive1.yaml"
```

File: assets/queries/dockerCompose/healthcheck_not_set/test/positive_expected_result.json [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+    },
+    {
+      "queryName": "Healthcheck Not Set",
+      "severity": "MEDIUM",
+      "line": 14,
+      "filename": "positive2.yaml"
```
Comment (joaorufi) [inline assets/queries/dockerCompose/healthcheck_not_set/test/positive_expected_result.json:12]:
```suggestion
      "fileName": "positive2.yaml"
```

File: assets/queries/dockerCompose/healthcheck_not_set/test/positive_expected_result.json [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+    },
+    {
+      "queryName": "Healthcheck Not Set",
+      "severity": "MEDIUM",
+      "line": 14,
+      "filename": "positive3.yaml"
```
Comment (joaorufi) [inline assets/queries/dockerCompose/healthcheck_not_set/test/positive_expected_result.json:18]:
```suggestion
      "fileName": "positive3.yaml"
```

**Review by joaorufi — CHANGES_REQUESTED** [review]

**Review by joaorufi — APPROVED** [review]

**Pull request merged**